package.json

```diff
@@ -5,15 +5,17 @@
   "main": "index.js",
   "scripts": {
     "dev": "pm2 start process-dev.json",
+    "build": "tsc",
+    "postinstall": "npm run test & npm run build",
+    "watch": "tsc -w",
+    "test": "eslint .",
     "docker:clean": "npm run docker:stop && ./bin/clean-up-docker-containers.sh",
     "docker:start": "DB_PORT=3002 ./bin/start-docker.sh",
     "docker:stop": "./bin/stop-docker.sh",
     "e2e-dev": "API_ENV=dev jest --rootDir ./tests/e2e ",
     "e2e": "API_ENV=prod jest --rootDir ./tests/e2e ",
     "local": "DB_PORT=3002 npm run docker:start && DB_PORT=3002 NODE_ENV=development node app.js",
```

Comment:
I think we should add [ts-node](https://www.npmjs.com/package/ts-node). if we do can we update this script to use it?